MQ2Camera.cpp

```diff
@@ -65,7 +66,7 @@ void SetCameraDistance(float distance)
 		float userValue = std::max(distance, s_origUserCameraMaxDistance);
 		SetCameraValue(UserCameraMaxDistance, userValue);
 
-		WriteChatf(PLUGIN_MSG "Camera max distance set to \ay%.2f\ax", distance);
+		WriteChatf("%s Camera max distance set to \ay%.2f\ax", PLUGIN_MSG, distance);
```

Comment:
The whole purpose of using the #define was to take advantage of the string literal concatenation. I explicitly did not want to have to use `%s` replacements.

Reply:
constexpr change reverted.  Let me know if it's cleaner to submit a new PR.